src/Directory.Build.targets

```diff
@@ -0,0 +1,23 @@
+<!-- This target must be imported into Directory.Build.targets -->
+<!-- Workaround. Remove once we're on 3.1.300+
+https://github.com/dotnet/sourcelink/issues/572 -->
```

Comment:
I will delete this file with the next coverlet release. The problem is already fixed in master.

Reply:
(`master` of coverlet)

Reply:
yes

Reply:
deleted